setup.py

```diff
@@ -40,7 +40,8 @@ def read(*names, **kwargs):
         'docopt==0.6.*',
         'xxhash==1.4.*',
         'agrc-usaddress==0.6.*',
-        'beautifulsoup4==4.8.*'
+        'beautifulsoup4==4.8.*',
+        'agrc-supervisor'
```

Comment:
**thought**: does this need a version or is this expecting a local install?

Reply:
I don't think so, but I'll tag @jacobdadams to help answer this one.

Reply:
We should add a major version here.

Reply:
Probably something like `agrc-supervisor==1.*`?

Reply:
yup. 1.* would work well since the [current version](https://pypi.org/project/agrc-supervisor/) is 1.1.0.

Reply:
Done